openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/AddIngestion.component.tsx

```diff
@@ -394,6 +394,8 @@ const AddIngestion = ({
       }
     } else if (showDBTConfig && activeIngestionStep === 2) {
       nextStep = activeIngestionStep + 2;
+    } else if (showDBTConfig && activeIngestionStep === 2) {
```

Comment:
@chirag-madlani got these changes from the UI after merging https://github.com/open-metadata/OpenMetadata/pull/8679 in this branch and merging `main`. IDK if this is correct. Please validate

Reply:
@pmbrull looks like the else if condition in both `handlePrev` and `handleNext` functions got repeated while merging. If you can remove the duplication that would be great

Reply:
thanks, solved!